### DISCUSSION TITLE
Asset relation based on which site group you uploaded it in

### DISCUSSION THREAD

**Discussion by Antimated** [craftcms/cms#12778] [Ideas] [answered]
Trying to add a logo in the CMS based on the site group. I have 2 site groups with 5 languages each. I need to be able to upload 1 logo for each site group.

Currently you are only able to manage asset relations on a per-site basis at most:
![Screenshot 2023-03-03 at 11 47 00](https://user-images.githubusercontent.com/2869409/222701178-2978cf50-29ed-49eb-a215-9791c5acef7c.png)

Translating fields based on site groups is already possible. e.g. the color field:

![Screenshot 2023-03-03 at 11 46 16](https://user-images.githubusercontent.com/2869409/222700863-ae500ea5-6bc9-4ff6-a4c5-781ae598feda.png)

**Comment by brandonkelly** [comment, marked as answer]
Thanks for posting! This was requested earlier via #8539, so going to mark this as a duplicate.
